test/cypress/integration/worldpay-3ds-flex-ddc/payment-with-session-id.spec.js

```diff
@@ -134,4 +216,15 @@ describe('Worldpay 3ds flex card payment flow', () => {
       })
     })
   })
+
+  setUpAndCheckCardPaymentPageWithGooglePay()
+  describe('Web payment text is not displayed on DDC spinner screen', () => {
+    it('iframe post should still submit the form but without the worldpaySessionId', () => {
+      cy.task('setupStubs', confirmPaymentDetailsStubs)
+      cy.get('#card-details').submit()
+
+      cy.get('.google-pay-container').should('have.attr', 'style', 'display: none;')
+      cy.get('.govuk-heading-l.web-payment-button-heading').should('have.attr', 'style', 'display: none;')
+    })
+  })
```

Comment:
Would it be worth having an Apple Pay equivalent of this?

Reply:
I did think about this but given that the code to hide Apple Pay is identical to Google Pay the test should be sufficient to verify whether worldpay-3ds-flex-ddc.js is functioning as intended. Furthermore given that we are going to reconceptualise the hiding we may refactor this test to simply check whether a `<div class="hidden-stuff>` is hidden/visible so seems redundant to add another test.